native-build-agent/Dockerfile

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2024 Eclipse Foundation and others.
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License 2.0
+# which is available at http://www.eclipse.org/legal/epl-v20.html
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+##
+# Installs required build tools that will be used in the compilation process
+# of fullstack development environments. This includes set and tested minimum
+# versions of node.js, Hugo, and is based on a GraalVM Mandrel image for native
+# compilation for Quarkus.
+##
+
+ARG MANDREL_VERSION=23.0
+FROM registry.access.redhat.com/quarkus/mandrel-23-rhel8:${MANDREL_VERSION}
+
+ARG NODE_VERSION=v18.20.2
+ARG NPM_VERSION=9.6.6
+ARG YARN_VERSION=1.22.19
+ARG MVN_VERSION=3.8.8
```

Comment:
Is there a good reason to use an old Maven version, instead of the latest one (3.9.6)?